### PR TITLE
[4.1] s/$presetButtonClases/$presetButtonClasses

### PR DESCRIPTION
**Pull request by PhilETaylor**
code review 

s/$presetButtonClases/$presetButtonClasses

edit: and short array syntax

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2a8f7eeca01cbc3227f24d70917844f98a229175<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36336">issues.joomla.org/tracker/joomla-cms/36336</a>.</sub>

File: layouts/plugins/editors/tinymce/field/tinymcebuilder.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
 			<joomla-tab-element class="tab-pane" id="set-<?php echo $num; ?>" <?php echo $isActive; ?> name="<?php echo $title; ?>">
 				<?php // Render tab content for each set ?>
 					<?php
-						$presetButtonClases = array(
+						$presetButtonClasses = array(
```
Comment (alikon) [inline layouts/plugins/editors/tinymce/field/tinymcebuilder.php]: shouldn't this become `[]` instead of `array()`

**Review by alikon — CHANGES_REQUESTED** [review]

File: layouts/plugins/editors/tinymce/field/tinymcebuilder.php (hunk 1 above)
Comment (PhilETaylor) [reply layouts/plugins/editors/tinymce/field/tinymcebuilder.php]: Correct.

Comment (PhilETaylor) [reply layouts/plugins/editors/tinymce/field/tinymcebuilder.php]: The joy of creating PRs in iOS Github App :-(

**Review by alikon — APPROVED** [review]

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 196f4ecc0dac4a4f02e8d365efeb25d9ae1d1538<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36336">issues.joomla.org/tracker/joomla-cms/36336</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36336">issues.joomla.org/tracker/joomla-cms/36336</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**